### PR TITLE
Add many-to-one between proposals and seconds

### PR DESCRIPTION
**Pull request by markdavison**
- Add missing relationship between proposals and seconds
- Rename references to seconds to be democracy seconds - breaking change

### CODE REVIEW

File: prawns/governance/schema.graphql [hunk 1]
```diff
@@ -22,7 +22,7 @@ type SubstrateGovernanceAccount @entity {
   totalTreasurySpendProposals: Int!
   councilVotes: [SubstrateCouncilVote]! @derivedFrom(field: "account")
   proposalVotes: [SubstrateProposalVote]! @derivedFrom(field: "account")
-  proposalSeconds: [SubstrateProposalSecond]! @derivedFrom(field: "account")
+  democracyProposalSeconds: [SubstrateDemocracyProposalSecond]! @derivedFrom(field: "account")
```
Comment (jdomingos) [inline prawns/governance/schema.graphql:25]: Regarding the renames here, should we actually duplicate the fields, having room to deploy these changes, let the clients migrate (native app) and then do another PR where we do the cleanup?

Comment (jdomingos) [reply prawns/governance/schema.graphql:25]: Just thinking, maybe just synching with them is just more practical but we need to accept some downtime :)

**Review by jdomingos — COMMENTED** [review]
Looks good @markdavison - just added a suggestion about the migration/deploy since you have a breaking change.

**Review by straticJeff — APPROVED** [review]

File: prawns/governance/schema.graphql (hunk 1 above)
Comment (jdomingos) [reply prawns/governance/schema.graphql:25]: But they are wrapped in the Graph:
https://graph.litentry.io/graphql?query=query%20MyQuery%20%7B%0AsubstrateDemocracyProposals%7B%0A%20%20account%20%7B%0A%20%20%20%20totalProposalSeconds%0A%20%20%20%20proposalSeconds%7B%0A%20%20%20%20%20%20account%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%7D

☝️ When you mean directly you also mean: "not even from the graph"?

Comment (franjs) [reply prawns/governance/schema.graphql:25]: I mean directly from the mobile apps.. The only data we are fetching from the mobile apps that is coming from the indexed data are the tips, and we are doing it with a custom resolver, so no schema stitching

**Review by jdomingos — APPROVED** [review]

**Comment by markdavison** [comment]
Have changed the base from main to a feature branch then will release it with all of democracy proposals requirements.
